build warns of deprecated classes marked for removal

The build currently warns of deprecated classes that are marked for removal and thus should be corrected soon.
```java
16:28:15  /jenkins/workspace/Adapters/IntelliJ/intellij-quarkus-build/src/main/java/com/redhat/devtools/intellij/lsp4ij/AbstractLSPInlayProvider.java:94: warning: [removal] getSimpleContext(String,Object,DataContext) in SimpleDataContext has been deprecated and marked for removal
16:28:15                  DataContext context = SimpleDataContext.getSimpleContext(LSP_COMMAND.getName(), command, DataManager.getInstance().getDataContext(source));
16:28:15                                                         ^
16:28:17  /jenkins/workspace/Adapters/IntelliJ/intellij-quarkus-build/src/main/java/com/redhat/devtools/intellij/quarkus/lang/QuarkusImplicitPropertyUsageProvider.java:17: warning: [removal] ImplicitPropertyUsageProvider in com.intellij.codeInspection.unused has been deprecated and marked for removal
16:28:17  public class QuarkusImplicitPropertyUsageProvider extends ImplicitPropertyUsageProvider {
16:28:17                                                            ^
```

--- a/src/main/java/com/redhat/devtools/intellij/lsp4ij/AbstractLSPInlayProvider.java
+++ b/src/main/java/com/redhat/devtools/intellij/lsp4ij/AbstractLSPInlayProvider.java
@@ -33,6 +33,7 @@ import org.jetbrains.annotations.Nullable;
 
 import javax.swing.JComponent;
 import java.awt.Component;
+import java.util.Map;
 
 public abstract class AbstractLSPInlayProvider implements InlayHintsProvider<NoSettings> {
     public static final DataKey<Command> LSP_COMMAND = DataKey.create("com.redhat.devtools.intellij.quarkus.lsp4ij.command");
@@ -91,7 +92,7 @@ public abstract class AbstractLSPInlayProvider implements InlayHintsProvider<NoS
         if (command != null) {
             AnAction action = ActionManager.getInstance().getAction(command.getCommand());
             if (action != null) {
-                DataContext context = SimpleDataContext.getSimpleContext(LSP_COMMAND.getName(), command, DataManager.getInstance().getDataContext(source));
+               DataContext context = SimpleDataContext.getSimpleContext(DataKey.create(LSP_COMMAND.getName()), command, DataManager.getInstance().getDataContext(source));
                 action.actionPerformed(new AnActionEvent(null, context,
                         ActionPlaces.UNKNOWN, new Presentation(),
                         ActionManager.getInstance(), 0));

--- a/src/main/java/com/redhat/devtools/intellij/quarkus/lang/QuarkusImplicitPropertyUsageProvider.java
+++ b/src/main/java/com/redhat/devtools/intellij/quarkus/lang/QuarkusImplicitPropertyUsageProvider.java
@@ -10,11 +10,11 @@
  ******************************************************************************/
 package com.redhat.devtools.intellij.quarkus.lang;
 
-import com.intellij.codeInspection.unused.ImplicitPropertyUsageProvider;
+import com.intellij.lang.properties.codeInspection.unused.ImplicitPropertyUsageProvider;
 import com.intellij.lang.properties.psi.Property;
 import com.redhat.devtools.intellij.quarkus.QuarkusModuleUtil;
 
-public class QuarkusImplicitPropertyUsageProvider extends ImplicitPropertyUsageProvider {
+public class QuarkusImplicitPropertyUsageProvider implements ImplicitPropertyUsageProvider {
     @Override
     public boolean isUsed(Property property) {
         return QuarkusModuleUtil.isQuarkusPropertiesFile(property.getContainingFile().getVirtualFile(), property.getProject());

